### PR TITLE
nos3#88 - cFS Upgrade 

### PR DESCRIPTION
**Pull request by jlucas9**
Updates to conform with draco-rc4

### CODE REVIEW

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c [hunk 1]
```diff
@@ -57,55 +57,55 @@ uint32 Ut_CFE_SB_GetMsgQueueDepth(void)
...
     UtListNode_t        *CurrentNode;
-    CFE_SB_Msg_t        *MessagePtr;
+    CFE_MSG_Message_t        *MessagePtr;
     uint32               MessageCount = 0;
 
     CurrentNode = UtList_First(&MsgQueue);
     while (CurrentNode) {
         MessagePtr = CurrentNode->Data;
-        if (MessageID == CFE_SB_GetMsgId(MessagePtr)) {
+        if (MessageID == CFE_MSG_GetMsgId(MessagePtr), CFE_SB_MsgId_t *MsgId) {
```
Comment (msuder) [inline fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: This looks wrong.

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c [hunk 2]
```diff
@@ -57,55 +57,55 @@ uint32 Ut_CFE_SB_GetMsgQueueDepth(void)
...
     UtListNode_t        *CurrentNode;
-    CFE_SB_Msg_t        *MessagePtr;
+    CFE_MSG_Message_t        *MessagePtr;
 
     CurrentNode = UtList_First(&MsgQueue);
     while (CurrentNode) {
         MessagePtr = CurrentNode->Data;
-        if (MessageID == CFE_SB_GetMsgId(MessagePtr)) {
-            return(TRUE);
+        if (MessageID == CFE_MSG_GetMsgId(MessagePtr), CFE_SB_MsgId_t *MsgId) {
```
Comment (msuder) [inline fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: This looks wrong.

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c [hunk 3]
```diff
@@ -57,55 +57,55 @@ uint32 Ut_CFE_SB_GetMsgQueueDepth(void)
...
     UtListNode_t        *CurrentNode;
-    CFE_SB_Msg_t        *MessagePtr;
+    CFE_MSG_Message_t        *MessagePtr;
     uint32               MessageCount = 0;
 
     CurrentNode = UtList_First(&MsgQueue);
     while (CurrentNode) {
         MessagePtr = CurrentNode->Data;
-        if (MessageID == CFE_SB_GetMsgId(MessagePtr)) {
+        if (MessageID == CFE_MSG_GetMsgId(MessagePtr), CFE_SB_MsgId_t *MsgId) {
```
Comment (msuder) [inline fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: This looks wrong.

**Review by msuder — APPROVED** [review]

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c (hunk 1 above)
Comment (jlucas9) [reply fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: Good catch! Should be resolved in https://github.com/nasa-itc/CFS_IO_LIB/pull/1/commits/8448e7e8807c3f83f48e942fd9d7ee37ea35b0cf

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c (hunk 2 above)
Comment (jlucas9) [reply fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: Good catch! Should be resolved in https://github.com/nasa-itc/CFS_IO_LIB/pull/1/commits/8448e7e8807c3f83f48e942fd9d7ee37ea35b0cf

File: fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c (hunk 3 above)
Comment (jlucas9) [reply fsw/unit_test/ut-assert/src/ut_cfe_sb_hooks.c]: Good catch! Should be resolved in https://github.com/nasa-itc/CFS_IO_LIB/pull/1/commits/8448e7e8807c3f83f48e942fd9d7ee37ea35b0cf

**Pull request merged**